src/modules/simulator/simulator_mavlink.cpp

```diff
@@ -95,6 +95,12 @@ void Simulator::pack_actuator_message(mavlink_hil_controls_t &actuator_msg) {
 		}
 	}
 
+	// if vehicle status has not yet been updated, set actuator commands to zero
+	// this is to prevent the simulation getting into a bad state
+	if (_vehicle_status.timestamp == 0) {
+		memset(out, 0, sizeof(out));
```

Comment:
Because the code above alters the values in out. The only other thing I could do is to put the if statement for the vehicle status around the code which puts values into out.

if (_vehicle_status.timestamp != 0) {
// set the outputs
}